application/event_processor/event_processor.py

```diff
@@ -190,6 +191,30 @@ def update_changes_with_opening_times(changes: dict, dos_service: DoSService, nh
         changes[OPENING_DAYS_KEY] = nhs_std_open_dates.export_cr_format()
 
 
+def log_unmatched_nhsuk_pharmacies(nhs_entity: NHSEntity) -> None:
+    """Log unmatched NHS Pharmacies
+    Args:
+        nhs_entity (NHSEntity): NHS entity to log
+    """
+    logger.info(
```

Comment:
Maybe a logger.warning is more appropriate?

Reply:
Sure